src/main/java/no/ssb/subsetsservice/SubsetsController.java

```diff
@@ -65,18 +70,23 @@ private void updateLDSURL(){
             LDSConsumer consumer = new LDSConsumer(LDS_SUBSET_API);
             ResponseEntity<JsonNode> ldsResponse = consumer.getFrom("/"+id);
             if (ldsResponse.getStatusCodeValue() == 404)
-                return consumer.postTo("/" + id, subsetJson);
+                return consumer.postTo("/" + id, Utils.cleanSubsetVersion(subsetJson));
         }
-        return new ResponseEntity<>(mapper.createObjectNode().put("error","Can not POST subset with id that is already in use. Use PUT to update existing subsets"), HttpStatus.BAD_REQUEST);
+        return ErrorHandler.newError("Can not POST subset with id that is already in use. Use PUT to update existing subsets", HttpStatus.BAD_REQUEST, LOG);
```

Comment:
Too long error message. Suggest "Cannot create a subset. The ID is already in use".